### PR TITLE
LibCpp: Add 4 new types

### PR DESCRIPTION
**Pull request by logkos**
Adds the following: ByteString, IPv6Address, Position, Token

### CODE REVIEW

File: Userland/Libraries/LibCpp/Lexer.cpp [hunk 1]
```diff
@@ -134,6 +134,7 @@ constexpr StringView s_known_types[] = {
     "Bitmap"sv,
     "ByteBuffer"sv,
     "Bytes"sv,
+    "ByteString"sv,
```
Comment (timschumi) [inline Userland/Libraries/LibCpp/Lexer.cpp:137]: :yakthonk:

Comment (logkos) [reply Userland/Libraries/LibCpp/Lexer.cpp:137]: not anymore 👍

**Comment by AtkinsSJ** [comment]
This should be `LibCpp: ...` rather than `Lexer: ...`. We have a lot of lexers. :^)

**Comment by logkos** [comment]
> This should be `LibCpp: ...` rather than `Lexer: ...`. We have a lot of lexers. :^)

done

**Review by timschumi — APPROVED** [review]

**Pull request merged**